lib/active_merchant/billing/gateways/iveri.rb

```diff
@@ -191,7 +193,8 @@ def headers(post)
         {
           'Content-Type' => 'text/xml; charset=utf-8',
           'Content-Length' => post.size.to_s,
-          'SOAPAction' => 'http://iveri.com/Execute'
+          'SOAPAction' => 'http://iveri.com/Execute',
+          'API-Version' => fetch_version # Add version to headers
```

Comment:
Why are we adding API-Version?

Reply:
Let's not add this to the header. If you found documentation somewhere that says we should add version to the headers then that could create a separate ticket.